PROTO_tests/tests/compare_test.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+import pytest
+
+import arkouda as ak
+
+N = 1_000_000
+
+"""
+The CompareTest class encapsulates unit tests that compare the results of analogous
+method invocations against Numpy ndarrays and Arkouda pdarrays to ensure equivalent
+results are generated.
+"""
+
+
+class TestCompare:
+    def test_compare_arange(self):
+        # create np version
+        nArange = np.arange(N)
+        # create ak version
+        aArange = ak.arange(N)
+        assert nArange.all() == aArange.all()
+
+    def test_compare_linspace(self):
+        # create np version
+        a = np.linspace(10, 20, N)
+        # create ak version
+        b = ak.linspace(10, 20, N)
+        # print(a,b)
+        assert np.allclose(a, b.to_ndarray())
+
+    def test_compare_ones(self):
+        # create np version
+        nArray = np.ones(N)
+        # create ak version
+        aArray = ak.ones(N)
+        assert nArray.all() == aArray.to_ndarray().all()
```

Comment:
I think `ones` and `zeroes` returns a float by default so maybe we should use `allclose` for these cases. It shouldn't be a problem but I feel like it safer